DQMOffline/Trigger/python/DQMOffline_Trigger_cff.py

```diff
@@ -204,6 +206,18 @@
     particleNetMonitoringHLT          # HIG: monitoring of HLT PNET taggers (incl. comparisons to Offline PNET)
 )
 
+_offlineHLTSource4HLTMonitorPDPh2 = cms.Sequence(
+    dqmInfoHLTMon *
+    HLTtrackerphase2DQMSource *           # phase-2 IT and OT clusters
+    trackingMonitorHLT *                  # tracking
+    hltToOfflineTrackValidatorSequence *  # Relative Online to Offline performace
+    vertexingMonitorHLT                   # vertexing
+)
+
+# remove Strip HLT monitoring in the phase-2 sequence
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
```

Comment:
`phase2_common` is not more appropriate ?
However, it should give the same result as `phase2_tracker` is always in phase-2 workflow.

Reply:
Well at the moment, it's fully tracker specific, so I'd rather not change it. In case it gets updated you can make sure the appropriate modifier is used

Reply:
OK, thx.